client/rest-high-level/src/test/java/org/opensearch/client/SearchIT.java

```diff
@@ -567,12 +567,12 @@ public void testSearchWithMatrixStats() throws IOException {
         MatrixStats matrixStats = searchResponse.getAggregations().get("agg1");
         assertEquals(5, matrixStats.getFieldCount("num"));
         assertEquals(56d, matrixStats.getMean("num"), 0d);
-        assertEquals(1830.0000000000002, matrixStats.getVariance("num"), 0d);
+        assertEquals(1830.0000000000002, matrixStats.getVariance("num"), 1.0e-12);
         assertEquals(0.09340198804973039, matrixStats.getSkewness("num"), 0d);
         assertEquals(1.2741646510794589, matrixStats.getKurtosis("num"), 0d);
         assertEquals(5, matrixStats.getFieldCount("num2"));
         assertEquals(29d, matrixStats.getMean("num2"), 0d);
-        assertEquals(330d, matrixStats.getVariance("num2"), 0d);
+        assertEquals(330d, matrixStats.getVariance("num2"), 1.0e-12);
         assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 1.0e-16);
         assertEquals(1.3517561983471071, matrixStats.getKurtosis("num2"), 0d);
         assertEquals(-767.5, matrixStats.getCovariance("num", "num2"), 0d);
```

Comment:
Shouldn't all the other `0d` tolerances be adjusted as well?

Reply:
We only observed variance become flaky for now. We can do the same thing for other metrics if we see same thing.

Reply:
I'll approve in the interest of moving things along but let another maintainer merge if they agree with this.

In general I think requiring double-precision floating point exact matches is a bad practice.  Even `Double.MIN_VALUE` as an epsilon is better.

Reply:
> In general I think requiring double-precision floating point exact matches is a bad practice

I tend to agree, but I'm going to merge this as it will solve a problem we're facing now. The other comparisons have been stable so we can address them if they show up as problematic.